### PR TITLE
Creating the contributing file

### PR DESCRIPTION
**Pull request by rjmajma**
This is the first look at the contributing file.

We'll likely revisit the team processes section in the future, but this works for now.

### CODE REVIEW

File: contributing [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+## How to Contribute
+
+This project constitutes a collaborative work ("open source"). Federal employees and members of the public are encouraged to improve the project by contributing. For more information, please see [the FAQ](https://project-open-data.cio.gov/faq/)
+
+Contributions can be made, primarily in two ways:
+
+### Easy
+
+1. Browse the content at https://project-open-data.cio.gov/.
```
Comment (ascott1) [inline contributing]: Should we change this URL? We could temporarily use this GitHub repo until it lives elsewhere

Comment (rjmajma) [inline contributing]: Hahah, good catch.

File: contributing [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Contributing to Project Open Data
+
+*By contributing to this project, you dedicate your work to the public domain and relinquish any copyright claims under the terms of the [CC0 Public Domain Dedication](http://creativecommons.org/publicdomain/zero/1.0/). See the [license file](LICENSE) for additional information.*
+
+## How to Contribute
+
+This project constitutes a collaborative work ("open source"). Federal employees and members of the public are encouraged to improve the project by contributing. For more information, please see [the FAQ](https://project-open-data.cio.gov/faq/)
```
Comment (ascott1) [inline contributing]: I didn't catch this earlier, but there's one more.

File: contributing [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Contributing to Project Open Data
```
Comment (Scotchester) [inline contributing]: Should say "Project Open Source".

Comment (rjmajma) [inline contributing]: I believe I made that change earlier. https://github.com/cfpb/project-open-source/commit/adbb0af950f5c93e0970fe00d1625cc3d0f48b4e

**Comment by Scotchester** [comment]
Looks good, now! :+1:

**Comment by ascott1** [comment]
:+1: Thanks @rjmajma!

**Comment by Scotchester** [comment]
Oops, one more thing: The file should be named `CONTRIBUTING.md`

**Pull request merged**